pages/api/mail/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ResponseUtil } from "@utils/responseUtil";
+import type { NextApiRequest, NextApiResponse } from "next";
+import send from "@utils/sendMail";
+import findEmails from "@database/mail";
+
+/**
+ * Handles the request to /api/mail (POST)
+ * @param req request object
+ * @param res response object
+ * @returns Promise<void> and sends json object back to the client (type: ResponseObject)
+ */
+export default async function mailHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "POST" && req.body.key == process.env.AWS_LAMBDA_KEY) {
+        // AWS_LAMBDA_SECRET
+        const result3h = await findEmails(3);
+        const result48h = await findEmails(48);
+        for (let i = 0; i < result3h.length; ++i) {
+            for (let j = 0; j < result3h[i].parentRegs.length; ++j) {
+                await send(
```

Comment:
Thoughts on using Promise.all for handling bulk email sending?

https://github.com/uwblueprint/dancefest-web/blob/6dbe38f8d950e4cf5c05911acb04752596d6838a/pages/api/feedback/index.js#L94-L96

Reply:
Done